src/main/java/by/mifort/automation/hr/dev/entity/MergeCandidates.java

```diff
@@ -0,0 +1,65 @@
+package by.mifort.automation.hr.dev.entity;
+
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+import org.apache.commons.lang3.builder.ToStringBuilder;
+
+import javax.persistence.Column;
+import javax.persistence.Embeddable;
+import java.io.Serializable;
+
+/**
+ * People merge primary keys
+ * @see CandidateMergeCandidates connected attributes with id
+ * @author yauheni_vozny
+ * @version 1.0
+ */
+
+@Embeddable
+public class MergeCandidates implements Serializable {
```

Comment:
why you need additional entity?

Reply:
It should be something like here https://stackoverflow.com/questions/51650881/two-onetomany-sets-to-same-table-in-hibernate